src/bin/vip-config-envvar.js

```diff
@@ -2,12 +2,42 @@
 
 import command from '../lib/cli/command';
 
+const usage = 'vip config envvar';
+const exampleUsage = 'vip @example-app.develop config envvar';
+
+// Command examples
+const examples = [
+	{
+		usage: `${ exampleUsage } delete MY_VARIABLE`,
+		description: 'Delete the environment variable "MY_VARIABLE" from the environment.',
+	},
+	{
+		usage: `${ exampleUsage } get-all`,
+		description: 'Retrieve a list of all environment variables in the default table format.',
+	},
+	{
+		usage: `${ exampleUsage } get MY_VARIABLE`,
+		description: 'Retrieve the value of the environment variable "MY_VARIABLE".',
+	},
```

Comment:
I mentioned the order on another, so if it's not a concern, please ignore! But these examples are also in the opposite order of the commands.